OpenRA.Mods.Common/Warheads/CreateEffectWarhead.cs

```diff
@@ -48,6 +48,9 @@ public class CreateEffectWarhead : Warhead
 		[Desc("Whether to consider actors in determining whether the explosion should happen. If false, only terrain will be considered.")]
 		public readonly bool ImpactActors = true;
 
+		[Desc("Ignore actors without any enabled Targetables even if ImpactActors is set to 'true' (instead of considering them as 'invalid target').")]
+		public readonly bool IgnoreUntargetableActors = false;
```

Comment:
I didn't just want to make that call on my own, but in my opinion a case could be made for this to default to 'true' instead.
It is a rare edge case, and it's more likely that 'true' will give the desired result, because that would make the warhead treat actors that don't have Targetables just like actors that don't have a Health trait.